src/tools/illink/test/Mono.Linker.Tests.Cases/DataFlow/FeatureCheckDataFlowTestSubstitutions.xml

```diff
@@ -0,0 +1,8 @@
+<linker>
+  <assembly fullname="test, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
+    <type fullname="ILLink.RoslynAnalyzer.TestFeatures">
```

Comment:
ILLink should allow specifying a substitution for CoreLib in the main app's assembly - if I remember correctly there are only two assemblies which are allowed to specify substitutions for "other" assembly - CoreLib and the main app's assembly (and command line substitutions obviously).
So for ILLink we could either add a substitutions for `IsDynamicCodeSupported` here by targeting CoreLib in the XML, or we could pass it as a command line parameter and separate xml file.

Reply:
OK - makes sense... I'm not asking to change anything, this looks good.